src/trw/train/sequence_array_fixed_samples_per_epoch.py

```diff
@@ -0,0 +1,96 @@
+import warnings
+
+import trw
+import trw.utils
+from trw.train import sequence
+from trw.train import sampler as sampler_trw
+import numpy as np
+import collections
+import copy
+from trw.utils import get_batch_n
+from trw.train import SequenceArray
+
+
+# this the name used for the sample UID
+sample_uid_name = 'sample_uid'
```

Comment:
use the global default uid name:
from trw.train.sequence_array import sample_uid_name as default_sample_uid_name